app/backend/graphql/customer/queries/customerQueryField.ts

```diff
@@ -0,0 +1,17 @@
+import { idArg, queryField } from 'nexus'
+import { isTeamMember } from '../../team/utils'
+import { Customer } from '../customer'
+
+export const customerQueryField = queryField('customer', {
```

Comment:
This just gets the title for a customer with a given id. What we need is a query that returns all customers for a given team but I guess this is part of the team object.